db/migrate/20141215175503_rename_via_script.rb

```diff
@@ -0,0 +1,285 @@
+    execute "ALTER TABLE ONLY oops_responses_symbols DROP CONSTRAINT entities_oops_responses_oops_response_id_fk;"
+    execute "ALTER TABLE ONLY oops_responses_symbols ADD CONSTRAINT oops_responses_symbols_oops_response_id_fk FOREIGN KEY (oops_response_id) REFERENCES oops_responses(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY sentences_symbols DROP CONSTRAINT entities_sentences_entity_id_fk;"
+    execute "ALTER TABLE ONLY sentences_symbols ADD CONSTRAINT sentences_symbols_symbol_id_fk FOREIGN KEY (symbol_id) REFERENCES symbols(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY sentences_symbols DROP CONSTRAINT entities_sentences_ontology_id_fk;"
+    execute "ALTER TABLE ONLY sentences_symbols ADD CONSTRAINT sentences_symbols_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY sentences_symbols DROP CONSTRAINT entities_sentences_sentence_id_fk;"
+    execute "ALTER TABLE ONLY sentences_symbols ADD CONSTRAINT sentences_symbols_sentence_id_fk FOREIGN KEY (sentence_id) REFERENCES sentences(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY symbol_groups DROP CONSTRAINT entity_groups_ontology_id_fk;"
+    execute "ALTER TABLE ONLY symbol_groups ADD CONSTRAINT symbol_groups_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY symbol_mappings DROP CONSTRAINT entity_mappings_source_id_fk;"
+    execute "ALTER TABLE ONLY symbol_mappings ADD CONSTRAINT symbol_mappings_source_id_fk FOREIGN KEY (source_id) REFERENCES symbols(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY symbol_mappings DROP CONSTRAINT entity_mappings_target_id_fk;"
+    execute "ALTER TABLE ONLY symbol_mappings ADD CONSTRAINT symbol_mappings_target_id_fk FOREIGN KEY (target_id) REFERENCES symbols(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY mapping_versions DROP CONSTRAINT link_versions_link_id_fk;"
+    execute "ALTER TABLE ONLY mapping_versions ADD CONSTRAINT mapping_versions_mapping_id_fk FOREIGN KEY (mapping_id) REFERENCES mappings(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY mapping_versions DROP CONSTRAINT link_versions_source_id_fk;"
+    execute "ALTER TABLE ONLY mapping_versions ADD CONSTRAINT mapping_versions_source_id_fk FOREIGN KEY (source_id) REFERENCES ontology_versions(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY mapping_versions DROP CONSTRAINT link_versions_target_id_fk;"
+    execute "ALTER TABLE ONLY mapping_versions ADD CONSTRAINT mapping_versions_target_id_fk FOREIGN KEY (target_id) REFERENCES ontology_versions(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY mappings DROP CONSTRAINT links_ontology_id_fk;"
+    execute "ALTER TABLE ONLY mappings ADD CONSTRAINT mappings_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
```

Comment:
Metrics/LineLength: Line is too long. [149/80]<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.